sw/device/tests/dif/dif_plic_sanitytest.c

```diff
@@ -7,18 +7,19 @@
 #include "sw/device/lib/arch/device.h"
 #include "sw/device/lib/base/log.h"
 #include "sw/device/lib/base/mmio.h"
+#include "sw/device/lib/base/print.h"
```

Comment:
Where is this being used?

Reply:
Nothing. It's left over from the old code.